src/components/Project/ProjectDetail/RelatedProjectSlider/RelatedProjectSlider.module.scss

```diff
@@ -69,10 +80,67 @@
   }
 }
 
+.cardSingleProject {
+  width: 679px;
+  display: flex;
+  align-items: center;
+  flex-flow: column wrap;
+  text-decoration: none;
+  
+
+  p {
+    font-size: var(--font-xxs);
+    font-weight: normal;
+    text-decoration: none;
+    text-align: center;
+  }
+
+  &:nth-of-type(1n + 2) {
+    margin-left: 12px;
+  }
+  
+  @media (min-width: 864px) {
```

Comment:
One thing I've found over time is, when working with SCSS, it's more maintainable to keep media queries "thin" and targetted.

What I mean is using multiple that are within their target selector, eg:

BEFORE:
```
  p {
    font-size: var(--font-xxs);
    font-weight: normal;
    text-decoration: none;
    text-align: center;
  }
  @media (min-width: 864px) {
    p {
      font-size: var(--font-small);
    }
  }
```

AFTER:
```
  p {
    font-size: var(--font-xxs);
    font-weight: normal;
    text-decoration: none;
    text-align: center;

    @media (min-width: 864px) {
        font-size: var(--font-small);
    }
  }
```

In more complex components this makes it much easier to look at a style and understand how it changes across breakpoints. You'll also find it makes it easier to change styles later down the line.